ui/packages/shared/profile/src/ProfileIcicleGraph/index.tsx

```diff
@@ -57,6 +64,46 @@ const ProfileIcicleGraph = ({
   const compareMode: boolean =
     selectQueryParam('compare_a') === 'true' && selectQueryParam('compare_b') === 'true';
   const {ref, dimensions} = useContainerDimensions();
+  const [storeSortBy, setStoreSortBy] = useURLState({param: 'sort_by', navigateTo});
+  const [localSortBy = FIELD_FUNCTION_NAME, setLocalSortBy] = useState(storeSortBy as string);
+  const setSortBy = useCallback(
+    (key: string) => {
+      setLocalSortBy(key);
+      setStoreSortBy(key);
+    },
+    [setLocalSortBy, setStoreSortBy]
+  );
+
+  const [storeGroupBy = [FIELD_FUNCTION_NAME], setStoreGroupBy] = useURLState({
+    param: 'group_by',
+    navigateTo,
+  });
+  const groupBy = useMemo(() => {
+    if (storeGroupBy !== undefined) {
+      if (typeof storeGroupBy === 'string') {
+        return [storeGroupBy];
+      }
+      return storeGroupBy as string[];
+    }
+    return [FIELD_FUNCTION_NAME];
+  }, [storeGroupBy]);
+
+  console.log('groupBy', storeGroupBy, groupBy);
```

Comment:
Looks like this log statement got slipped through.

Reply:
Oh, I thought I caught them all :smiley: